daisy/README.md

```diff
@@ -1,28 +1,141 @@
-# Daisy
+Definitions:
+* GCP: Google Cloud Platform
+* GCS: Google Cloud Storage
+* GCE: Google Compute Engine
+* Workflow: a graph of executable, blocking steps and their dependency
+relationships
 
-Daisy is a GCE workflow tool. Development is ongoing.
+# What is Daisy?
+Motivated by Google-internal image build workflows, Daisy is a solution for
+running complex, multi-step procedures on and involving GCE.
 https://godoc.org/github.com/GoogleCloudPlatform/compute-image-tools/daisy
 
-## Table of contents
+The current Daisy stepset includes support for creating/deleting GCE resources,
+waiting for signals from GCE VMs, streaming GCE VM logs, uploading local files
+to GCE and GCE VMs, and more.
+
+For example, Daisy is used to create Google Official Guest OS images. The
+workflow:
+1. Creates a Debian 8 disk and another empty disk.
+2. Creates and boots a VM with the two disks.
+3. Waits for the VM to run a script which installs an OS on the empty disk.
+4. Creates an image from the previously empty disk, now with an OS on it.
+5. Automatically cleans up ephemeral workflow resources (the VM and disks).
+
+Other use-case examples:
+* Image import workflows for images from other virtualization providers.
```

Comment:
Import virtual or physical images or disks from outside of GCE and run workflows on them to make these images work on GCE.